python/sparktk/frame/ops/categorical_summary.py

```diff
@@ -1,4 +1,181 @@
+from sparktk.propobj import PropertiesObject
 
-def categorical_summary(self, column_inputs):
-    # waiting for joyesh's jconvert updates, since the input here needs to be a list of dictionary (string to tuple)
-    raise NotImplementedError("Not implemented -- waiting for Joyesh's jconvert updates.")
\ No newline at end of file
+class CategoricalSummaryInput(PropertiesObject):
+    """
```

Comment:
Not sure I buy the need for an Input class --python users can just enter tuples...  move the validation to a function that takes an arbitrary tuple.  Open to discuss...